common/src/main/java/io/netty/util/internal/CleanerJava9.java

```diff
@@ -78,7 +87,7 @@ public void freeDirectBuffer(ByteBuffer buffer) {
         // See https://bugs.openjdk.java.net/browse/JDK-8191053.
         if (System.getSecurityManager() == null) {
             try {
-                INVOKE_CLEANER.invoke(PlatformDependent0.UNSAFE, buffer);
+                INVOKE_CLEANER_HANDLE.invokeExact(buffer);
```

Comment:
In the past I have had pain using invokeExact (if the types aren't exactly right, normal covariance doesn't work, or something like that).  

Consider using plain `invoke`.   I'm moderately confident the "exact" overload doesn't have any performance benefit.

Reply:
Tomorrow I will give a better look (still recovering from a fever today), but I have experienced the opposite ie invokeExact to get a huge benefit in perf if compared with just invoke. I need to re-check, but that was my past experience with it.

Reply:
@carl-mastrangelo I tested this one and it works... I would prefer to use `invokeExact` if possible.